src/HieDb/Types.hs

```diff
@@ -215,6 +215,17 @@ instance FromRow DefRow where
   fromRow = DefRow <$> field <*> field <*> field <*> field
                    <*> field <*> field
 
+data ExportRow = ExportRow
+  { exportHieFile :: FilePath
+  , exportName :: OccName
```

Comment:
Is the `OccName` really enough? Don't you need the `Module` as well to disambiguate between name conflicts in different modules?

Reply:
I didn't add the module name since you can always get it by doing a join with the `modules` table, but happy to add it if you think that's better

Reply:
I don't think you can? You can export identifiers defined in other modules

Reply:
the `exportHieFile` tells us which module the identifer was *exported from*, but doesn't tell us which module it was *defined in*.

Reply:
But we are tracking exports here, not the "defined in" property, right? I think I might be missing your point.

Reply:
`Data.Map` and `Data.Set` both export different versions of `singleton`. If I try to look up which module exports `Data.Map.singleton` with the current schema, won't I get false positives for `Data.Set.singleton`?

Reply:
I don't understand the question, you already know which module exports `Data.Map.singleton`.

If you wanted to lookup which modules export `singleton`, you'd need a query `SELECT exports.*,mods.mod FROM exports JOIN mods USING hieFile`.

Reply:
> I don't understand the question, you already know which module exports Data.Map.singleton.

Another module can re-export it - the project might not have an explicit dependency on `containers` and we might want to import it from an internal module other than `Data.Map` which re-exports `singleton`.

Reply:
I still don't understand the problem. 

```
sqlite> SELECT exports.*,mods.mod FROM exports JOIN mods USING (hieFile) WHERE occ = 'vlogDebug';
/Users/pepeiborra/.cache/ghcide/ghcide-1.4.0.2-inplace-95ed415e96c426ac060c7580b3dd10092e2471c1/Development/IDE/Types/Logger.hie|vlogDebug||0|Development.IDE.Types.Logger
/Users/pepeiborra/.cache/ghcide/ghcide-1.4.0.2-inplace-95ed415e96c426ac060c7580b3dd10092e2471c1/Development/IDE.hie|vlogDebug||0|Development.IDE
```

Reply:
It is fine as long as there is only one underlying `logDebug`. If there were two `logDebug`s being used from different libraries, then there is ambiguity.

My point is that the exports in the hie file capture information about the exact `Name` that was exported. However, we lose some of this information in the table, keeping only the `OccName` instead of enough information to reconstruct the `Name`.

This might be OK for what `ghcide` wants to do now, but suppose you want to gather statistics or generate a graph/visualization of exports from your project. In that scenario it would be useful to have this extra information.

Reply:
Thanks for clarifying! I finally see your point. "mod" column name now added to the exports table.

Reply:
We need to record a `UnitId` as well - module names aren't enough to disambiguate (`PackageImports`) !

The parent can itself have a different `ModuleName` and `UnitId`. This is the case for data families, where the data family type and instances can be defined in different modules. Either those need to be recorded as well, or the parent field needs to be a reference into the table itself.

Reply:
Added unit as well

Reply:
Added parentMod and parentUnit